### ISSUE TITLE
[iOS, React Native v0.71.4, RNAA v6.4.3] TypeError: Cannot read property 'authorize' of undefined

### ISSUE THREAD

**Issue by aauffray** [FormidableLabs/react-native-app-auth#864]
## Issue

React Native v0.71.4
react-native-app-auth v6.4.3
Running on  Simulator, iOS v16.4

Works well on Android
On iOS, I always get the `TypeError: Cannot read property 'authorize' of undefined` error when calling `authorize`. I tried importing with `import AppAuth from 'react-native-app-auth'` and `import { authorize } from 'react-native-app-auth'`, I get the same error with both. My guess is that there's a linking problem somewhere.

Pods are installed, I see the `Installing AppAuth (1.4.0)` line in green when installing.

Here is my `AppDelegate.h` :

```
#import <RCTAppDelegate.h>
#import <UIKit/UIKit.h>
#import <Firebase.h>
#import "RNAppAuthAuthorizationFlowManager.h"

@interface AppDelegate : RCTAppDelegate <RNAppAuthAuthorizationFlowManager>

@property(nonatomic, weak) id<RNAppAuthAuthorizationFlowManagerDelegate> authorizationFlowManagerDelegate;

@end
```

And the `AppDelegate.mm`:

```
#import "AppDelegate.h"
#import <Firebase.h>

#import <React/RCTBundleURLProvider.h>

#import <React/RCTLinkingManager.h>

#import "RNAppAuthAuthorizationFlowManager.h" 

@implementation AppDelegate

- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
{
  [FIRApp configure];
  self.moduleName = @"MY_APP_NAME";
  self.initialProps = @{};

  return [super application:application didFinishLaunchingWithOptions:launchOptions];
}

- (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
{
#if DEBUG
  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index"];
#else
  return [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
#endif
}

- (BOOL)concurrentRootEnabled
{
  return true;
}

- (BOOL)application:(UIApplication *)app
            openURL:(NSURL *)url
            options:(NSDictionary<NSString *, id> *)options
{
  if ([self.authorizationFlowManagerDelegate resumeExternalUserAgentFlowWithURL:url]) {
    return YES;
  }
  return [RCTLinkingManager application:app openURL:url options:options];
}

@end
```

---

## Environment

* **Your Identity Provider**: Azure AD B2C
* **Platform that you're experiencing the issue on**: `iOS`
* **Are you using Expo?** No

**Comment by ArshanKhanifar** [comment]
Has anybody actually made this work with React Native 0.71.x

The library's documentation seemingly has no support for it, and their example app [uses a much older version of RN](https://github.com/FormidableLabs/react-native-app-auth/blob/484ec68646233f50f238520da8dd3f2c32f41fa6/Example/package.json#LL14C14-L14C14).

**Comment by airimia1995** [comment]
+1

**Comment by abdi4** [comment]
+1

**Comment by tolidevs** [comment]
+1

**Comment by fukemy** [comment]
I am using this lib since version 0.6x, now it still working in version 0.71.8

**Comment by lewisd1996** [comment]
@fukemy Can you share your `AppDelegate.h` and `AppDelegate.mm` files?

**Comment by robwalkerco** [comment]
Fixed by #900
